client/state/action-types.js

```diff
@@ -249,6 +249,8 @@ export const HELP_TICKET_CONFIGURATION_REQUEST_FAILURE =
 export const HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS =
 	'HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS';
 export const HTTP_REQUEST = 'HTTP_REQUEST';
+export const I18N_LOCALE_SUGGESTIONS_REQUEST = 'I18N_LOCALE_SUGGESTIONS_REQUEST';
+export const I18N_LOCALE_SUGGESTIONS_SUCCESS = 'I18N_LOCALE_SUGGESTIONS_SUCCESS';
```

Comment:
So imagine if we were creating a local app only. A more informative name might be something like: `I18N_LOCALE_SUGGESTIONS_ADD`

Reply:
also think about how we would describe it if we were just dumping this into state from the console

Reply:
Good suggestions, thanks. Just working on this simple-ish conversion, the clouds have really parted for me on the usefulness of the data layer. 😄